Makefile

```diff
@@ -125,6 +125,22 @@ fmt-check: required-gofmt-version
 test: fmt-check
 	go test $(PACKAGES)
 
+.PHONY: less-check
+less-check: less
+	@diff=$$(git diff); \
+	if [ -n "$$diff" ]; then \
+		echo "Please run 'make less' and commit the result:"; \
+		echo "$${diff}"; \
+		exit 1; \
+	fi;
+
+.PHONY: less
+less:
```

Comment:
It would be better to update `public/css/index.css` and `stylesheets` target to have compat with old syntax and be included in `assets`